modules/puzzle/src/main/Selector.scala

```diff
@@ -37,10 +37,16 @@ private[puzzle] final class Selector(
           .uno[Puzzle]
       case Some(user) if user.perfs.puzzle.nb >= maxAttempts => fuccess(none)
       case Some(user) =>
-        val rating = user.perfs.puzzle.intRating min 2300 max 900
-        val step = toleranceStepFor(rating)
-        api.attempt.playedIds(user) flatMap { ids =>
-          tryRange(rating, step, step, difficultyDecay(difficulty), ids, isMate)
+        val nextLearn = api.learning.nextPuzzle(user)
+        val isLearn = scala.util.Random.nextInt(5) == 0
```

Comment:
I imagined it would be easier to know if it exists before attempting to make a decision to use it, this is also a left over from when it was 50:50. I'll change it.